website/src/examples/terrain-extension.js

```diff
@@ -0,0 +1,95 @@
+import React, {Component} from 'react';
+import {MAPBOX_STYLES, GITHUB_TREE} from '../constants/defaults';
+import App from 'website-examples/terrain-extension/app';
+
+import {makeExample} from '../components';
+
+const LOCATIONS = {
+  'Stage 5': {
+    latitude: 43.09822,
+    longitude: -0.6194,
+    zoom: 10
+  },
+  'Stage 6': {
+    latitude: 43.02905,
+    longitude: 0.11472,
+    zoom: 10
+  },
+  'Stage 9': {
+    latitude: 45.82382,
+    longitude: 2.2899,
+    zoom: 8
+  },
+  'Stage 13': {
+    latitude: 46.07586,
+    longitude: 5.38004,
+    zoom: 9
+  },
+  'Stage 14': {
+    latitude: 45.75779,
+    longitude: 6.48885,
+    zoom: 9
+  },
+  'Stage 15': {
+    latitude: 45.90044,
+    longitude: 6.49025,
+    zoom: 9
+  },
+  'Stage 17': {
+    latitude: 45.89081,
+    longitude: 6.66961,
+    zoom: 9
+  },
+  'Stage 20': {
+    latitude: 47.8761,
+    longitude: 6.9999,
+    zoom: 10
+  }
+};
+
+class TerrainExtensionDemo extends Component {
+  static title = 'Tour de France routes';
+
+  static code = `${GITHUB_TREE}/examples/website/terrain-extension`;
+
+  static parameters = {
+    location: {
+      displayName: 'Location',
+      type: 'select',
+      options: Object.keys(LOCATIONS),
+      value: 'Stage 5'
+    }
+  };
+
+  static mapStyle = MAPBOX_STYLES.BLANK;
+
+  static renderInfo(meta) {
+    return (
+      <div>
+        <p>Reconstructed 3D terrain with routes from mapbox's Elevation service.</p>
```

Comment:
Describe the routes data here and link to source. You can remove the link to Mapbox since that's not the feature being showcased.